gammapy/catalog/fermi.py

```diff
@@ -56,22 +56,226 @@ class SourceCatalogObject3FGL(SourceCatalogObject):
+                                              )
+        ss += '{:<20s} : {:.4f} deg\n'.format('Semiminor (68%)', d['Conf_68_SemiMinor'])
+        ss += '{:<20s} : {:.2f} deg\n'.format('Position angle (68%)', d['Conf_68_PosAng'])
 
-        ss += 'RA (J2000)  : {}\n'.format(d['RAJ2000'])
-        ss += 'Dec (J2000) : {}\n'.format(d['DEJ2000'])
-        ss += 'GLON        : {}\n'.format(d['GLON'])
-        ss += 'GLAT        : {}\n'.format(d['GLAT'])
+        ss += '{:<20s} : {:.4f} deg\n'.format('Semimajor (95%)', d['Conf_95_SemiMajor'])
+        ss += '{:<20s} : {:.4f} deg\n'.format('Semiminor (95%)', d['Conf_95_SemiMinor'])
+        ss += '{:<20s} : {:.2f} deg\n'.format('Position angle (95%)', d['Conf_95_PosAng'])
+
+        return ss
+
+    def _info_spectral(self):
+        """Print spectral info."""
+        d = self.data
+        ss = '\n*** Spectral info ***\n\n'
+        ss += '{:<45s} : {:.3} +- {:.3} erg cm^-2 s^-1\n'.format('Energy flux (100 MeV - 100 GeV)', d['Energy_Flux100'],
+                                                                 d['Unc_Energy_Flux100'])
+        ss += '{:<45s} : {:.3f} Sigma\n'.format('Detection significance (100 MeV - 300 GeV)', d['Signif_Avg'])
+        ss += '{:<45s} : {}\n'.format('Spectrum type', d['SpectrumType'])
+        if d['SpectrumType'].rstrip() == 'LogParabola':
+            ss += '{:<45s} : {} +- {}\n'.format('beta', d['beta'], d['Unc_beta'])
+        if d['SpectrumType'].rstrip() in ['PLExpCutoff', 'PlSuperExpCutoff']:
+            ss += '{:<45s} : {:.0f} +- {:.0f} MeV\n'.format('Cutoff energy', d['Cutoff'], d['Unc_Cutoff'])
+        if d['SpectrumType'].rstrip() == 'PLSuperExpCutoff':
+            ss += '{:<45s} : {} +- {}\n'.format('Exponential index', d['Exp_Index'], d['Unc_Exp_Index'])
+        ss += '{:<45s} : {:.3f}\n'.format('Power law index', d['PowerLaw_Index'])
+        ss += '{:<45s} : {:.3f} +- {:.3f}\n'.format('Spectral index', d['Spectral_Index'], d['Unc_Spectral_Index'])
+        ss += '{:<45s} : {:.0f} MeV\n'.format('Pivot energy', d['Pivot_Energy'])
+        ss += '{:<45s} : {:.3} +- {:.3} cm^-2 MeV^-1 s^-1\n'.format('Flux Density (100 MeV - 100 GeV)',
+                                                                    d['Flux_Density'],
+                                                                    d['Unc_Flux_Density'])
+        ss += '{:<45s} : {:.3} +- {:.3} cm^-2 s^-1\n'.format('Integral flux (1 - 100 GeV)', d['Flux1000'],
+                                                             d['Unc_Flux1000'])
+        ss += '{:<45s} : {:.1f}\n'.format('Significance curvature', d['Signif_Curve'])
+
+        ss += '\n\n'
+        ss += 'Table:\n\n'
+        ss += '{:<15} {:<35} {:<25} {:<20}\n'.format('Energy range', 'Integral flux', 'Energy distribution',
+                                                     'Sqrt Test Statistic')
+        flux_table = {
+            # Omitting the 30-100 MeV range because it's not filled?
```

Comment:
Yes, it's not filled. They apparently decided on the catalog format before the mission, and then wanted to keep the format, even if they chose to make the catalog only above 100 MeV.
So you can remove the `?` and commented out part about 30-100 MeV here.